test/e2e/utils/utils.go

```diff
@@ -259,6 +259,32 @@ func TestWebhookServerTLS(ctx context.Context, f *framework.Framework, serviceNa
 	return nil
 }
 
+func TestPluginWithOperator(deviceName string, resourceNames []v1.ResourceName, f *framework.Framework, ns string, ctx context.Context, timeout time.Duration) {
+	operatorYaml := "deployments/operator/default/kustomization.yaml"
+	dpSampleYaml := "deployments/operator/samples/deviceplugin_v1_" + deviceName + "deviceplugin.yaml"
+
+	Kubectl("", "apply", "-k", operatorYaml)
+
+	if _, err := e2epod.WaitForPodsWithLabelRunningReady(ctx, f.ClientSet, ns, labels.Set{"control-plane": "controller-manager"}.AsSelector(), 1, timeout); err != nil {
+		framework.Failf("unable to wait for all pods to be running and ready: %v", err)
+	}
```

Comment:
why deploy the operator several times? if it's desired, it can also go to a `BeforeEach` or something, right?

Reply:
Umm, it makes sense to put it in `BeforeEach`, but it does not prevent operator to be deployed several times, because it runs before each ginkgo.It().

Reply:
anyway, done!